config/routes.rb

```diff
@@ -1,12 +1,14 @@
 Rails.application.routes.draw do
+  resources :events, only: [:new, :index, :show, :create, :attended_event, :unattend_event]
+  resources :users
+ 
   get 'events/new'
-  get 'events/index'
   get 'events/show'
-  resources :users
-  resources :events
-  
+  post 'add_events_attending', to: 'events#attended_event'
+  post 'rem_events_attending', to: 'events#unattend_event'
+  get '/signup', to: 'users#new'
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
```

Comment:
__OPTIONAL__: You could use resource check out [this link](https://guides.rubyonrails.org/routing.html)